Developed in-house, not fed back.
Solaris-specific: configuration file customization.

Note that server.set-v6only is disabled to enable
IPv4 connections by default, and recover the same
behaviour as in lighttpd 1.4.45 and older.

See the following webpage to get more information.
https://redmine.lighttpd.net/boards/2/topics/7720

--- lighttpd-1.4.82/doc/config/conf.d/fastcgi.conf
+++ lighttpd-1.4.82/doc/config/conf.d/fastcgi.conf
@@ -22,8 +22,8 @@ ##
 #fastcgi.server = ( ".php" =>
 #                   ( "php-local" =>
 #                     (
-#                       "socket" => socket_dir + "/php-fastcgi-1.socket",
-#                       "bin-path" => server_root + "/cgi-bin/php5",
+#                       "socket" => socket_dir + "/php-fastcgi.socket",
+#                       "bin-path" => "/usr/php/bin/php-cgi",
 #                       "max-procs" => 1,
 #                       "broken-scriptfilename" => "enable",
 #                     ),
--- lighttpd-1.4.82/doc/config/lighttpd.annotated.conf
+++ lighttpd-1.4.82/doc/config/lighttpd.annotated.conf
@@ -13,11 +13,11 @@ ##
 ## if you add a variable here. Add the corresponding variable in the
 ## chroot example as well.
 ##
-var.log_root    = "/var/log/lighttpd"
-var.server_root = "/srv/www"
-var.state_dir   = "/run"
-var.home_dir    = "/var/lib/lighttpd"
-var.conf_dir    = "/etc/lighttpd"
+var.log_root    = "/var/lighttpd/1.4/logs"
+var.server_root = "/var/lighttpd/1.4"
+var.state_dir   = log_root
+var.home_dir    = server_root
+var.conf_dir    = "/etc/lighttpd/1.4"
 
 ##
 ## run the server chrooted.
@@ -58,7 +58,7 @@ ##
 ## used in:
 ## conf.d/deflate.conf
 ##
-var.cache_dir   = "/var/cache/lighttpd"
+var.cache_dir   = server_root + "/cache"
 
 ##
 ## Base directory for sockets.
@@ -93,12 +93,15 @@ ## (default: "*" for all local IPv4 inte
 ##
 #server.bind = "localhost"
 
+# Enable IPv4 connections.
+server.set-v6only = "disable"
+
 ##
 ## Run as a different username/groupname.
 ## This requires root permissions during startup.
 ##
-server.username  = "lighttpd"
-server.groupname = "lighttpd"
+server.username  = "webservd"
+server.groupname = "webservd"
 
 ##
 ## Enable lighttpd to serve requests on sockets received from systemd
@@ -114,7 +117,7 @@ #server.core-files = "disable"
 ##
 ## Document root
 ##
-server.document-root = server_root + "/htdocs"
+server.document-root = server_root + "/docroot"
 
 ##
 ## The value for the "Server:" response field.
@@ -126,7 +129,7 @@ #server.tag = "lighttpd"
 ##
 ## store a pid file
 ##
-server.pid-file = state_dir + "/lighttpd.pid"
+server.pid-file = log_root + "/run.pid"
 
 ##
 #######################################################################
@@ -374,7 +377,7 @@ ##
 ## Format: <errorfile-prefix><status-code>.html
 ## -> ..../status-404.html for 'File not found'
 ##
-#server.errorfile-prefix    = server_root + "/htdocs/errors/status-"
+#server.errorfile-prefix    = server_root + "/errors/status-"
 
 ##
 ## mimetype mapping
--- lighttpd-1.4.82/doc/config/lighttpd.conf
+++ lighttpd-1.4.82/doc/config/lighttpd.conf
@@ -3,8 +3,8 @@ ##
 ## /etc/lighttpd/lighttpd.conf is the entry point for lighttpd configuration.
 ## This file may include others using 'include' or 'include_shell'.
 ##
-## Print configuration: lighttpd -f /etc/lighttpd/lighttpd.conf -p
-## Check configuration: lighttpd -f /etc/lighttpd/lighttpd.conf -tt
+## Print configuration: lighttpd -f /etc/lighttpd/1.4/lighttpd.conf -p
+## Check configuration: lighttpd -f /etc/lighttpd/1.4/lighttpd.conf -tt
 ##
 ## References
 ## Wiki: https://wiki.lighttpd.net/
@@ -13,9 +13,9 @@ ## Configuration: File Syntax: https://w
 ## Configuration: Options: https://wiki.lighttpd.net/Docs_ConfigurationOptions
 ##
 ## There are multiple ways to configure lighttpd; no one-size-fits-all method.
-## 1. single file; everything in /etc/lighttpd/lighttpd.conf
-## 2. include specific files from /etc/lighttpd/conf.d/...
-## 3. include all files from /etc/lighttpd/conf-enabled/*
+## 1. single file; everything in /etc/lighttpd/1.4/lighttpd.conf
+## 2. include specific files from /etc/lighttpd/1.4/conf.d/...
+## 3. include all files from /etc/lighttpd/1.4/conf-enabled/*
 ## 4. generate config from script (e.g. using 'include_shell')
 ## 5. combination of one or more of the above, or some other way ...
 ## Please choose what works best for you.
@@ -31,12 +31,12 @@ ##     /etc/lighttpd/lighttpd.annotated.
 ##     and comments describing those directives.  More details may be found
 ##     by following the links in the References section above.
 ##
-include "/etc/lighttpd/lighttpd*annotated.conf"
+include "/etc/lighttpd/1.4/lighttpd*annotated.conf"
 ##
 ## (3) may be implemented via the following (and ok if files do not exist)
 ##
-include "/etc/lighttpd/conf-enabled/*.conf"
+include "/etc/lighttpd/1.4/conf-enabled/*.conf"
 ##
 #######################################################################
 
-#server.document-root = "/var/www/html"
+#server.document-root = "/var/lighttpd/1.4/docroot"
